src/mlpack/methods/ann/layer/dropout.hpp

```diff
@@ -47,21 +47,17 @@ namespace ann /** Artificial Neural Network. */ {
  * @tparam OutputDataType Type of the output data (arma::colvec, arma::mat,
  *         arma::sp_mat or arma::cube).
  */
-template <
-    typename InputDataType = arma::mat,
-    typename OutputDataType = arma::mat
->
+template<typename InputDataType = arma::mat,
+    typename OutputDataType = arma::mat>
```

Comment:
Can you line up the `typename OutputDataType` with the `typename InputDataType`?  I think it is a little more readable that way.  (I'm sorry, I know it is a pedantic comment.)